docs/api-guide/viewsets.md

```diff
@@ -213,14 +214,14 @@ Note that the `basename` is provided by the router during `ViewSet` registration
 
 Using the example from the previous section:
 
-```python
->>> view.reverse_action('set-password', args=['1'])
+```pycon
```

Comment:
Is pycon a valid syntax highlighter?

Reply:
Yes:
https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer